libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -197,4 +201,51 @@
   .feds-promo--dark a:hover {
     text-decoration: underline;
   }
+
+
+  .feds-links-strip-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    border-top: solid 1px #f3f3f3;
+  }
+
+  .feds-links-strip {
+    width: 100%;
+    padding: 20px 0;
+    max-width: var(--feds-maxWidth-nav);
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .feds-links-strip > *:not(:last-child)::after {
+    content: '|';
+    margin: 0;
+  }
+
+  .feds-links-strip-item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .feds-popup .feds-navLink:hover {
+    background-color: transparent;
+  }
+
+  .feds-links-strip-image {
+    & picture {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    & img {
+      width: 25px;
+      object-fit: contain;
+    }
+  }
```

Comment:
This is LESS/SASS syntax and achieves nothing, this whole section could probably be removed.

Reply:
The `&` nesting selector is also valid CSS syntax. I used it to handle the image coming from the `*.docx`, but I'll probably need to remove this if I switch to svgs

Reply:
Regarding SVGs, I think I misunderstood the intent. For now, it should be fine to keep just .jpg/.png support. Hopefully we can re-use the linkGroup decoration logic so that when we add SVG support there, it trickles down here as well.

Regarding the CSS nesting - I still wouldn't use it right now, as browsers with full support have started rolling it out in October/November this year. I'd wait a bit before commiting to using it